batchlib/analysis/cell_analysis_qc.py

```diff
@@ -317,10 +318,20 @@ def outlier_heuristics(self, in_files):
             for name, val in ratios.items():
                 if not name.startswith('ratio'):
                     continue
-                if val < 0.:
+                if val < 0. or np.isnan(val):
                     is_outlier = 1
                     outlier_type += f'{name} is negative'
 
+        min_infected_intensity = self.outlier_criteria['min_infected_intensity']
+        if min_infected_intensity is not None:
+            # TODO is this the correct stat?
+            intensity_key = f'{self.serum_key}_means'
+            # TODO do we take the median ?
+            infected_intensity = np.median(infected_stats[intensity_key])
```

Comment:
@imagirom @tischi Do we take the median over the infected cells here or something else?